source/release-notes/index-4x.rst

```diff
@@ -77,7 +77,6 @@ Wazuh version                                  Release date
 
    .. toctree::
 
-      4.11.1 Release notes <release-4-11-1>
       4.11.0 Release notes <release-4-11-0>
       4.10.2 Release notes <release-4-10-2>
```

Comment:
We've removed `release-4-4-10-2.rst` document and references. The upwards merge should reflect it here.